### PR TITLE
Grid with videos

### LINKED ISSUE #221: Add Pattern: Grid with videos
🛠️ The goal of this issue is to track the implementation of the Grid with videos pattern.
🧑‍💻 The implementation must follow the [pattern creation guidelines](https://github.com/WordPress/twentytwentyfive/blob/trunk/README.md#pattern-creation-guidelines), and should be assigned to its appropriate category.

[Figma](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=3177-2028&m=dev)

For the "Podcast" text, please use a paragraph block with the "pill" style applied.
You can use empty placeholders for the video blocks.

image

### PR DESCRIPTION
**Pull request by eirichmond**
**Description**
First attempt of Grid with videos, I'm not sure about tablet view but they are simply placeholders.

**Screenshots**

https://github.com/user-attachments/assets/d61f0e2f-a55c-4482-ac71-5fcda684a435

**Testing Instructions**
Create page
Add 'Grid with videos' pattern from banner section
Test desktop, tablet and mobile

### CODE REVIEW

File: patterns/grid-videos.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+		<figure class="wp-block-video"></figure>
+		<!-- /wp:video -->
+	</div>
+	<!-- /wp:group -->
+</div>
+<!-- /wp:group -->
+
+<!-- wp:paragraph -->
+<p></p>
+<!-- /wp:paragraph -->
```
Comment (carolinan) [inline patterns/grid-videos.php]: Please remove the stray paragraph, and leave a single blank line at the end of the file.

**Comment by carolinan** [comment]
Hi
I'm sorry for the wait, I have been reviewing the oldest pull requests first. Please change the first group block, the outmost group, to full width instead of wide.

File: patterns/grid-videos.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+ * Categories: banner, about, featured
+ * Description: A grid with videos.
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:group {"metadata":{"name":"Grid with videos"},"align":"wide","style":{"spacing":{"padding":{"top":"var:preset|spacing|50","bottom":"var:preset|spacing|50","left":"var:preset|spacing|50","right":"var:preset|spacing|50"},"blockGap":"var:preset|spacing|60"}},"layout":{"type":"constrained"}} -->
```
Comment (carolinan) [inline patterns/grid-videos.php]: please remove the metadata.

**Comment by beafialho** [comment]
Thank you for your work on this one!

> Please change the first group block, the outmost group, to full width instead of wide.

This is also my only comment. Everything else looks good.

File: patterns/grid-videos.php [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+		<!-- wp:video -->
+		<figure class="wp-block-video"></figure>
+		<!-- /wp:video -->
+	</div>
+	<!-- /wp:group -->
+</div>
+<!-- /wp:group -->
+
+<!-- wp:paragraph -->
+
```
Comment (carolinan) [inline patterns/grid-videos.php]: This paragraph needs to be fully removed and there should be a single blank line at the end of the file after the closing `<!-- /wp:group -->`

Comment (eirichmond) [reply patterns/grid-videos.php]: @carolinan is there an example of the black line you can point me to in the figma designs? I can't seem to find one or at least the one I can see seems to be part of the footer.

Comment (carolinan) [reply patterns/grid-videos.php]: "black line" was a typo :) already corrected to blank line.

Comment (carolinan) [reply patterns/grid-videos.php]: I am talking about leaving a blank line at the end of the file, not "at the bottom of the design".

File: patterns/grid-videos.php [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+		<!-- /wp:video -->
+
+		<!-- wp:video -->
+		<figure class="wp-block-video"></figure>
+		<!-- /wp:video -->
+	</div>
+	<!-- /wp:group -->
+</div>
+<!-- /wp:group -->
+
```
Comment (carolinan) [inline patterns/grid-videos.php]:
```suggestion
```

**Comment by carolinan** [comment]
Thank you

**Pull request merged**